flutter/lib/src/default_integrations.dart

```diff
@@ -76,6 +81,14 @@ class FlutterErrorIntegration extends Integration<SentryFlutterOptions> {
 
     options.sdk.addIntegration('flutterErrorIntegration');
   }
+
+  @override
+  void close() {
+    /// Restore default
+    FlutterError.onError = defaultOnError;
```

Comment:
Should check if `FlutterError.onError` still holds a reference to the callback you set within this class. If someone else wraps `onError` after this runs, you'd be removing them too.